resources/js/src/app/shared/UrlBuilder.ts

```diff
@@ -16,12 +16,22 @@ class UrlBuilder {
 
         urlParams = urlParams.split('+').join(' ');
         while ((tokens = regex.exec(urlParams)) !== null) {
-            params[decodeURIComponent(tokens[1])] = decodeURIComponent(tokens[2]);
+            params[decodeURIComponent(tokens[1])] = this.sanitizeHTML(decodeURIComponent(tokens[2]));
         }
 
         return params;
     }
 
+    sanitizeHTML(str : string): string {
```

Comment:
⛏️ It's a 1:1 duplication from the code above. Use the same function in both cases.

Reply:
Removed from the other class, that class wasnt even used